problems/core/list.py

```diff
@@ -0,0 +1,4 @@
+class ListNode:
```

Comment:
I don't like this file naming, may be structure.py or data.py

Reply:
I would like to have all entities separated, all in individual files.